udata-front-plugins-helper/package.json

```diff
@@ -1,23 +1,35 @@
 {
   "name": "@etalab/udata-front-plugins-helper",
-  "version": "1.0.2-0",
+  "version": "1.0.2-7",
   "description": "Ready-made helper for plugins to integrate them in udata-front.",
   "author": "Etalab",
-  "main": "dist/index.js",
+  "main": "src/index.js",
+  "types": "types/index.d.ts",
   "files": [
-    "dist",
-    "src"
+    "src",
+    "types"
   ],
   "scripts": {
-    "build": "tsc",
-    "preversion": "npm run build",
+    "build": "vue-tsc",
+    "test": "vitest",
```

Comment:
I choose `vitest` here because it's made for unit tests whereas `Cypress` is for end-to-end tests and visual component tests.
We can add Cypress later for component tests with a real browser but I think end-to-end tests will stay in udata-front root because they require the udata CLI and server / test database.
The added benefit is that it uses `vite` so we have the same configuration format here and in udata-front root.